### ISSUE TITLE
How about some ml-based method?

### ISSUE THREAD

**Issue by Thewillman** [ASC-Competition/ASC24-LLM-inference-optimization#5]
such as I use a pretrained ml-based model as a scheduler to make the whole LLM system reach balance， or use other neural network to make model early exit？
